### PR TITLE
Design: web based audio player initial design

### PR DESCRIPTION
**Pull request by tusharwalzade216**
This PR includes all the assignments from Week 1 -
1. The document for the information about browser functionality

2. the design for a web based audio player.
Screenshot

3. The document for the summary of product mindset and design thinking

### CODE REVIEW

File: Week-1/Assignment-2/index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<html>
```
Comment (sairamin) [inline Week-1/Assignment-2/index.html:2]: Where is DOCTYPE ?

Comment (sairamin) [inline Week-1/Assignment-2/index.html:2]: Alignment please

File: Week-1/Assignment-2/index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            <li><button><em class="las la-step-backward"></em></button></li>
+            <li><button><em class="las la-play"></em></button></li>
+            <li><button><em class="las la-step-forward"></em></button></li>
+            <li><button><em class="las la-undo-alt"></em></button></li>
+            <li><button><em class="las la-random"></em></button></li>
+        </ul>
+    </div>
+</footer>
+
+</html>
```
Comment (sairamin) [inline Week-1/Assignment-2/index.html]: add a new line at end of each file

File: Week-1/Assignment-2/index.html (hunk 1 above)
Comment (tusharwalzade216) [reply Week-1/Assignment-2/index.html:2]: done

Comment (tusharwalzade216) [reply Week-1/Assignment-2/index.html:2]: added it

File: Week-1/Assignment-2/index.html (hunk 2 above)
Comment (tusharwalzade216) [reply Week-1/Assignment-2/index.html]: added a newline at the end

**Comment by tusharwalzade216** [comment]
Created separate branched & appropriate PRs for each assignment, so closing this long PR.

**Pull request closed without merging**